src/server/test.rs

```diff
@@ -138,7 +140,11 @@ fn collection_usage() {
 
 #[test]
 fn configuration() {
-    test_endpoint(http::Method::GET, "/42/info/configuration", "{}");
+    test_endpoint(
+        http::Method::GET,
+        "/42/info/configuration",
+        &serde_json::to_string(&ServerLimits::default()).unwrap(),
```

Comment:
I did wonder whether this test is too tautological to be useful, but I think it's okay. Asserting that the response payload equals the JSON serialization of the default `ServerLimits` is fine, yeah?